c-dependencies/js-compute-runtime/xqd.wit

```diff
@@ -1,4 +1,4 @@
-world xqd-world {
+default world xqd-world {
```

Comment:
What's the effect of this change?

Reply:
The world files were upgraded with a bunch of new semantics including packages and use support. Default worlds and default interfaces are the new way of defining the item in the wit document to use by default since they can contain multiple worlds (otherwise you need to explicitly pass `--world` to specify which world is being generated out of the file).